tutorials/23_claim_rewards/index.py

```diff
@@ -10,11 +10,11 @@
 }
 
 #capture user information
-username = input('Enter username: ') #demo account: cdemo
-wif = input('Enter private ACTIVE key: ') #demo account: 5KaNM84WWSqzwKzY82fXPaUW43idbLnPqf5SfjGxLfw6eV2kAP3
+username = input('Enter username: ') #demo account: demo01
+wif = input('Enter private ACTIVE key: ') #demo account: 5HxTntgeoLm4trnTz94YBsY6MpAap1qRVXEKsU5n1v2du1gAgVH
```

Comment:
Not sure why we're using the active key for this.  The `client.claim_reward_balance` operation only requires the posting key.

Reply:
We have used the ACTIVE key for all operations on the TESTNET as far as I recall. I will merge but if you would prefer to use the POSTING We can load it as a bug and fix it.